yente/search/search.py

```diff
@@ -85,9 +85,29 @@ async def search_entities(
     aggregations: Optional[Dict[str, Any]] = None,
     sort: List[Any] = [],
 ) -> Dict[str, Any]:
+    # Wrap query in function_score to up-score important entities.
+    function_score_query = {
+        "function_score": {
+            "query": query,
+            "functions": [
+                {
+                    "field_value_factor": {
+                        "field": "num_values",
+                        # This is a bit of a jiggle factor. Currently, very large documents (like Vladimir Putin)
+                        # have a num_values of ~200, so get a +10 boost.
+                        # The order is modifier(factor * value)
+                        "factor": 0.5,
+                        "modifier": "sqrt",
+                    }
+                }
+            ],
+            "boost_mode": "sum",
```

Comment:
so 

1 prop value ends up doing subquery_score + 0.71
2 prop values end up doing subquery_score + 1
3 prop values end up doing subquery_score + 1.2

what sort of range would subquery_score be for a search for something like `John Smith`? How big an impact does it make for cases where lots of candidates with very similar scores?

I'm exploring what the chance is of us excluding candidates with very few props but that match decently, for things that elastic scores similarly but for whom a boost of 1 would be a big deal.

Reply:
So *the* Obama will get a +7.4, *the* Putin a +10.

Reply:
ok - I think the relative size between subquery_Score and what we're adding is sane